GeneratorInterface/ThePEGInterface/plugins/ThePEGHadronizer.cc

```diff
@@ -41,15 +49,22 @@ class ThePEGHadronizer : public ThePEGInterface, public gen::BaseHadronizer {
 	const char *classname() const { return "ThePEGHadronizer"; }
 
     private:
+
+        virtual void doSetRandomEngine(CLHEP::HepRandomEngine* v) override { setPEGRandomEngine(v); }
```

Comment:
I will remove this line and hope it fixes our problem with the compilation.

In the worst case this could lead to using the ThePEG random number generator with the same initial seed for each job. But this should be no problem since we will use it only for showering LHE files.